src/util.ts

```diff
@@ -313,3 +314,13 @@ export type XapkManifest = {
     expansions?: { file: string; install_location: string; install_path: string }[];
     split_apks?: { file: string; id: string }[];
 };
+
+export const parsePemCertificateFromFile = async (path: string) => {
```

Comment:
I implemented something similar in #82 :D Oh well, we’ll merge our definitions there.